### ISSUE TITLE
A strange error when installing robotjs

### ISSUE THREAD

**Issue by stevobm** [octalmage/robotjs#275]
I try to install robotjs but meet a strange error, is this a bug of robotjs or simply a problem of my environment? 
Here are the details:
```
G:\test>npm install robotjs

> robotjs@0.4.5 install G:\test\node_modules\robotjs
> prebuild --install

prebuild WARN install connect ETIMEDOUT 192.30.253.113:443
prebuild WARN install got an error, rolling back install
prebuild ERR! configure error
prebuild ERR! stack Error: tunneling socket could not be established, cause=Parse Error
prebuild ERR! stack     at ClientRequest.onError (G:\test\node_modules\tunnel-agent\index.js:176:17)
prebuild ERR! stack     at ClientRequest.g (events.js:273:16)
prebuild ERR! stack     at emitOne (events.js:90:13)
prebuild ERR! stack     at ClientRequest.emit (events.js:182:7)
prebuild ERR! stack     at Socket.socketOnData (_http_client.js:353:9)
prebuild ERR! stack     at emitOne (events.js:90:13)
prebuild ERR! stack     at Socket.emit (events.js:182:7)
prebuild ERR! stack     at readableAddChunk (_stream_readable.js:153:18)
prebuild ERR! stack     at Socket.Readable.push (_stream_readable.js:111:10)
prebuild ERR! stack     at TCP.onread (net.js:529:20)
prebuild ERR! not ok
prebuild ERR! build Error: tunneling socket could not be established, cause=Parse Error
prebuild ERR! build     at ClientRequest.onError (G:\test\node_modules\tunnel-agent\index.js:176:17)
prebuild ERR! build     at ClientRequest.g (events.js:273:16)
prebuild ERR! build     at emitOne (events.js:90:13)
prebuild ERR! build     at ClientRequest.emit (events.js:182:7)
prebuild ERR! build     at Socket.socketOnData (_http_client.js:353:9)
prebuild ERR! build     at emitOne (events.js:90:13)
prebuild ERR! build     at Socket.emit (events.js:182:7)
prebuild ERR! build     at readableAddChunk (_stream_readable.js:153:18)
prebuild ERR! build     at Socket.Readable.push (_stream_readable.js:111:10)
prebuild ERR! build     at TCP.onread (net.js:529:20)
npm WARN enoent ENOENT: no such file or directory, open 'G:\test\package.json'
npm WARN test No description
npm WARN test No repository field.
npm WARN test No README data
npm WARN test No license field.
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "D:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Administrator\\AppData\\Roaming\\npm\\node_modules\\npm\
\bin\\npm-cli.js" "install" "robotjs"
npm ERR! node v5.7.1
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE

npm ERR! robotjs@0.4.5 install: `prebuild --install`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the robotjs@0.4.5 install script 'prebuild --install'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the robotjs package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     prebuild --install
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs robotjs
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls robotjs
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     G:\test\npm-debug.log
```

Besides, if I only want to use the binary of robotjs, is it necessary to install python?

**Comment by OmriSoudry** [comment]
Having the same problem, Can't install robotJS.

**Comment by GeorgeBgk** [comment]
Yep

**Comment by octalmage** [comment]
This should be resolved by v0.4.6, please let me know if you still encounter this and we can reopen the issue.
